cmd/integration/commands/stages.go

```diff
@@ -412,7 +412,7 @@ func stageSenders(db kv.RwDB, ctx context.Context) error {
 	s := stage(sync, tx, nil, stages.Senders)
 	log.Info("Stage", "name", s.ID, "progress", s.BlockNumber)
 
-	cfg := stagedsync.StageSendersCfg(db, chainConfig, tmpdir)
+	cfg := stagedsync.StageSendersCfg(db, chainConfig, tmpdir, prune.Mode{})
```

Comment:
set real one plz (get it from DB).

Reply:
as far as stage senders is regarded pruning does not come into place in stage senders since we need senders to still generate the state. I think that putting at least in tests a void prune.Mode would be more clear to read than taking it out of the database.

Reply:
same as tests, let me know what you think